dspace-spring-rest/src/main/webapp/login.html

```diff
@@ -79,6 +79,26 @@ <h2 class="form-signin-heading">HAL Browser</h2>
             "onclick" : function() { toastr.remove(); }
         }
 
+        $.ajax({
+          url : window.location.href.replace("login.html", "") + 'api/authn/login',
+          success : function(result, status, xhr) {
+              document.cookie = "MyHalBrowserToken=" + xhr.getResponseHeader('Authorization').split(" ")[1];
+              toastr.success('You are now logged in. Please wait while we redirect you...', 'Login Successful');
+              setTimeout(function() {
+                  window.location.href = window.location.pathname.replace("login.html", "");
+              }, 2000);
```

Comment:
It looks like this is a copy of the `success` handler below. So it would be better to group this logic into a separate function. It would be something like this (untested): 

```
var successHandler = function(result, status, xhr) {
              document.cookie = "MyHalBrowserToken=" + xhr.getResponseHeader('Authorization').split(" ")[1];
              toastr.success('You are now logged in. Please wait while we redirect you...', 'Login Successful');
              setTimeout(function() {
                  window.location.href = window.location.pathname.replace("login.html", "");
              }, 2000);
}

$.ajax({
          url : window.location.href.replace("login.html", "") + 'api/authn/login',
          success : successHandler
}

...

$("#login").click(function() {
            $.ajax({
                  ...
                 success: successHandler
    })
 })
```